core/templates/components/summary-tile/story-summary-tile.component.html

```diff
@@ -29,7 +29,16 @@
   <!-- Left-to-right is enforced here due to the formatting of the HTML with the title on the left and chapter count on the right. -->
   <div class="oppia-story-title" dir="ltr">
     <strong class="title-text">
-      <a [href]="storyLink" [ngClass]="{'disabled-link': !classroomUrlFragment}">{{ storyTitle }} </a>
+      <a [href]="storyLink"
+         [ngClass]="{'disabled-link': !classroomUrlFragment}"
+         *ngIf="!storyTitleTranslationKeyIsToBeDisplayed">
+         {{ storyTitle }}
+      </a>
+      <a [href]="storyLink"
+         [ngClass]="{'disabled-link': !classroomUrlFragment}"
+         *ngIf="storyTitleTranslationKeyIsToBeDisplayed">
```

Comment:
Just to check -- to reduce the repetition in the branching, does putting a span (with the ngIfs on that) inside the `a` element work?

Reply:
Added it inside a span